src/common/components/ReviewItem/ReviewItem.theme.ts

```diff
@@ -46,23 +49,14 @@ export const reviewItemTheme = tv(
         "text-ellipsis",
         "text-text-em-low",
       ],
-      body: [
-        "relative",
-        "flex",
-        "h-16",
-        "w-full",
-        "self-stretch",
-        "overflow-hidden",
-        "text-sm",
-        "text-ellipsis",
-        "text-text-em-high",
-      ],
+      body: ["h-16", "text-sm", "text-text-em-high", "line-clamp-3"],

```

Comment:
note: somehow `line-clamp` doesnt seem to work well here _(note the lingering charaters at the 4th line)_, perhaps it has to do with `line-height`s?

![image](https://github.com/AfterClass-io/afterclass.io-v2/assets/13061926/59523eba-f5b9-4b5b-b97c-ca6638e25f98)

Reply:
fixed height of review body. this issue doesnt show up on locally for me so let me know if this fixes the issue
https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/5f4e39304d70be3a1557d6c8e5ef2785b42a965d

Reply:
Maybe we shouldn't hard code the height. There should be another way to do it automatically! Let us know if you can't get it to work without setting the height

Reply:
removing the fixed height works, fixed https://github.com/AfterClass-io/afterclass.io-v2/pull/104/commits/bd72536b78788ff4f2d0eade4163c9980907ed17

Reply:
confirmed, thanks!